src/components/StyleGuide/Filtering/PageCompositionElements/ActiveFilters.mdx

```diff
@@ -0,0 +1,65 @@
+import { Meta } from "@storybook/blocks";
+
+import Image from "../../../Image/Image";
+import Link from "../../../Link/Link";
+
+import imageFilteringActiveFiltersList from "/static/filteringStyleGuide/filteringActiveFiltersList.png";
+import { exampleWrapperStyles } from "../Overview.mdx";
+
+<Meta title="Style Guide/Filtering/Page elements/Active filters list" />
+
+# Active filters list
+
+The `active filters` list is used to display filters that have been applied.
+
+<Image
+  alt="Example of active filters list"
+  src={imageFilteringActiveFiltersList}
+  {...exampleWrapperStyles}
+/>
+
+## Table of Contents
+
+- {<Link href="#rendering" target="_self">Rendering</Link>}
+- {<Link href="#position" target="_self">Position</Link>}
+- {<Link href="#visibility" target="_self">Visibility</Link>}
+- {<Link href="#labeling" target="_self">Labeling</Link>}
+- {<Link href="#adding-filters" target="_self">Adding filters</Link>}
+- {<Link href="#clearing-filters" target="_self">Clearing filters</Link>}
+
+## Rendering
+
+The `active filters` list should be rendered using the DS
+[TagSet](./?path=/docs/components-content-display-tagset--docs) component.
+
+## Position
+
+The `active filters` list should be positioned directly above the `results` list.
+
+## Visibility
+
+The `active filters` list should only be visible after at least one filter option
+has been applied.
+
+## Labeling
+
+The `active filters` list should include an "Active filters" label.
+
+## Adding filters
+
+A new tag should be added to the `TagSet` as each new filter option is applied.
+The tags in the `TagSet` should be listed in the order they were selected.
```

Comment:
Is there something built into the TagSet that would maintain that order?

Reply:
The tags within the `TagSet` component are rendered using the order of the elements in the data object that is passed in. It will be the responsibility of consuming apps to handle the order of the elements in that data object.